README.md

```diff
@@ -71,7 +71,21 @@ cd /workspace
 git clone https://github.com/CiscoTestAutomation/xpresso
 ```
 
-**2. Initializations**
+**2. Initializing using setup script**
+
+**NOTE:** If you want to do manual setup , then you can skip this step.
+
+Run the script with providing the proper URL of the server.
+
+```
+./setup.sh  http://youmachinehostname/
+
+```
+
```

Comment:
It seems we're either missing the warning that is present in [this PR](https://github.com/CiscoTestAutomation/xpresso/pull/30) in this readme, or we need to add this check to the setup script.

Reply:
Added in script to check  if  "/"   is in the end of string .